contracts/DAO.sol

```diff
@@ -41,7 +41,10 @@ contract DAO {
     uint256 public trg;//total rewards generated
     mapping(address=>uint256) public lrc;//last rewards claimed at trg point for this address
     mapping(address=>uint256) public lST;//last stake time
-
+    mapping(address=>uint256) public lastClaimPoint;//point at which above a user is eligable to claim
+    mapping(uint256=>uint256) public claimPoints;//points of ETH transfers to this contract
+    mapping(uint256=>uint256) public totalStakedAtClaimPoint;//points of ETH transfers to this contract
```

Comment:
Are duplicate comments intentional?

Reply:
good point I've added a accurate comment for totalStakedAtClaimPoint